drivers/video/Kconfig

```diff
@@ -31,6 +31,21 @@ config VIDEO_BUFFER_POOL_ALIGN
 	int "Alignment of the video pool’s buffer"
 	default 64
 
+config VIDEO_BUFFER_USE_SHARED_MULTI_HEAP
+	bool "Use shared multi heap for video buffer"
+	default n
+
+config VIDEO_BUFFER_SMH_ATTRIBUTE
+	int "Shared multi heap attribute for video buffer"
+	depends on VIDEO_BUFFER_USE_SHARED_MULTI_HEAP
+	default 0
+	range 0 2
+	help
+	  Shared multi heap attribute for video buffer:
+	  0: SMH_REG_ATTR_CACHEABLE
+	  1: SMH_REG_ATTR_NON_CACHEABLE
+	  2: SMH_REG_ATTR_EXTERNAL
```

Comment:
It is also possible to make use of *choice* depending on tastes:
https://docs.zephyrproject.org/latest/build/kconfig/setting.html#configuring-choices

It is also possible to make them all optional, so that `VIDEO_BUFFER_USE_SHARED_MULTI_HEAP` and `VIDEO_BUFFER_SMH_ATTRIBUTE` can be merged together:
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#optional-choices

Not sure what is the best as I am still learning Zephyr best practices, maybe how you proposed is best!

Reply:
My idea here was to use `CONFIG_VIDEO_BUFFER_SMH_ATTRIBUTE` directly as an argument in `shared_multi_heap_aligned_alloc` based on the `SMH` enum:

```
enum shared_multi_heap_attr {
	/** cacheable */
	SMH_REG_ATTR_CACHEABLE,

	/** non-cacheable */
	SMH_REG_ATTR_NON_CACHEABLE,

	/** external Memory */
	SMH_REG_ATTR_EXTERNAL,

	/** must be the last item */
	SMH_REG_ATTR_NUM,
};
```

I think this might be cleaner than to have choices in Kconfig and have to use `#ifdefs`. What do you think?

Reply:
What you picked is the least bureaucratic! Thank you for clarifying, this also teaches me how to map Kconfig to C enums.